kedro-datasets/pyproject.toml

```diff
@@ -34,7 +34,7 @@ min-public-methods = 1
 [tool.coverage.report]
 fail_under = 100
 show_missing = true
-omit = ["tests/*", "kedro_datasets/holoviews/*"]
+omit = ["tests/*", "kedro_datasets/holoviews/*", "kedro_datasets/snowflake/*"]
```

Comment:
Why exclude?

Reply:
This was to exclude the execution of snowpark tests from the coverage report and avoid % coverage errors.

To execute snowpark tests we need a live connection to a snowflake account, we planned to keep it separated from the main tests and only execute when triggering tests with a snowflake marker (we added `make test-snowflake-only`). While the test execution is skipped if no snowflake marker is provided, it still affects the coverage report which expects 100% - which is why I added it to the omit from the report. 

Let us know your thoughts and if there's any other way around, and we can discuss further.

Reply:
No it's looking good - makes sense :)